rank_llm/rankllm.py

```diff
@@ -66,12 +59,23 @@ def permutation_pipeline(
         rank_end: int,
         logging: bool = False,
     ):
+        def filter_permutation(permutation, passage_count):
```

Comment:
this is not required since there are already two functions that take care of cleaning the permutations, plus this wouldn't work for gpt3.5 or gpt4 outputs